md/widgets.md

```diff
@@ -9,6 +9,36 @@ A page consists of widgets that are arranged in rows, and optionally, containers
 
 Each widget in the palette has a description. To show a widget description, just roll over the widget in the palette.
 
+## Widget Actions
+
+:::info
+ Not available for containers.
+:::
+
+Close to the widget name in the properties panel, several options are available in widgets actions button.
```

Comment:
"From the properties panel header, the "Widget actions" menu provides a list of actions for the selected widget"